web/yo/app/scripts/controllers/gene.js

```diff
@@ -454,11 +454,10 @@ angular.module('oncokbApp')
                 return result;
             }
             var userNames = [];
-            function prepareReviewItems() {
```

Comment:
This function is kinda complicated, we might need to do some refactoring for proper usage. For now, I'm just replace the scope status to function-wide variable. These three are not really used across genes page. The logic of `sectionUUIDs` seems not really straight forward since it has been used in another two methods. So I'm gonna just assign it to global sectionUUIDs at the place calls getReviewInfo.